pkg/ocm/helpers.go

```diff
@@ -720,7 +721,7 @@ func (c *Client) CheckUpgradeClusterVersion(
 		}
 	}
 	if !validVersion {
-		return weberr.Errorf(
+		return errors.Errorf(
```

Comment:
I think I got used to using and there were some places were it was in use already

Since there's no REST response it will act as a normal error anyways